data/duke.txt

```diff
@@ -0,0 +1,5 @@
+T | 0 | test
+T | 0 | test
+T | 0 | test
+T | 0 | test
+T | 0 | test
```

Comment:
I would recommend ignoring data/duke.txt file in .gitignore since this file most likely will change everytime you run the duke app. Therefore, it would be more convenient to just ignore this file.

Reply:
noted, thank you!